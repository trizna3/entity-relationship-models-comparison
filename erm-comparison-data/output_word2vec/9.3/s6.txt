students-EMPTY
contracts-contracts
buildings-buildings
payments-EMPTY
rooms-rooms
account-EMPTY
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-floor-rooms
