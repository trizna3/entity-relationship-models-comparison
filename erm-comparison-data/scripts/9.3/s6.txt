entity set buildings {
	"address"
};

entity set rooms {
	"capacity"
	"fee"
};

entity set contracts {
	"start"
	"end"
};

entity set accommodated {
};

association between 
{
	buildings ONE 
	rooms MANY
}{
	"floor"
};

association between
{
	rooms ONE 
	contracts MANY
}{};

association "payments" between 
{
	contracts MANY 
	accommodated ONE
}{};
