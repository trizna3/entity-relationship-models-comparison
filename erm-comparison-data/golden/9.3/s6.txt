buildings-buildings
rooms-rooms
contracts-contracts
payments-payments
students;account-accommodated
CONTRACT_11_ASSOCIATION:E-students-account
REBIND_MN_TO_1NN1:S-contracts-accommodated